manifests/plugin/ansible/runner.pp

```diff
@@ -0,0 +1,48 @@
+# = Ansible runner package installation
+#
+# $manage_runner_repo:: If true, adds upstream repositories to install ansible-runner package from
+#
+class foreman_proxy::plugin::ansible::runner(
+  Boolean $manage_runner_repo = $foreman_proxy::plugin::ansible::manage_runner_repo,
+) {
+
+  if $manage_runner_repo {
+    case $::osfamily {
+      'Debian': {
+        include ::apt
+        ::apt::source { 'ansible-runner':
```

Comment:
The leading colons are no longer needed. The rest of the module still has to be converted.

```suggestion
        apt::source { 'ansible-runner':
```